homeworks/roman.plynko_RPVanarx/homework_7/task1.test.js

```diff
@@ -0,0 +1,32 @@
+jest.mock('./getRandomNumber', () => ({
+    getRandomNumber: jest.fn(),
+}));
+
+const { getRandomNumber } = require('./getRandomNumber');
+const { doFiboArray } = require('./doFiboArray');
+const { isLess100, isBigger50 } = require('./checkRandomNumber');
+
+describe('tеst random 50 to 100', () => {
+    test('numbеr is biggеr 50', () => {
+        getRandomNumber.mockReturnValueOnce(49);
+        expect(isBigger50()).toBeFalsy();
```

Comment:
jest has various types of check, not only boolean like `toBeFalsy` or `toBeTruthy`

you could use equality checks like `toBe` and `toEqual`